app/controllers/seances_controller.rb

```diff
@@ -0,0 +1,26 @@
+class SeancesController < ApplicationController
+  # GET /seances or /seances.json
+  def index
+    @seances = Seance.includes(:movie, :hall).all
+  end
+  # GET /seances/new
+  def new
+    @seance = Seance.new
+    @halls = Hall.all
+    @movies = Movie.all
+  end
+  # POST /seances or /seances.json
+  def create
+    @seance = Seance.new(seance_params)
+
+    respond_to do |format|
+      if @seance.save
+        format.html { redirect_to seance_url(@seance), notice: "Seance was successfully created." }
+        format.json { render :show, status: :created, location: @seance }
```

Comment:
Do you need a response in JSON format?

Reply:
I don't need it now, so I will delete it. Thanks :D